### ISSUE TITLE
Clicking horizontal or vertical scrollbar trough changes table data sorting.

### ISSUE THREAD

**Issue by jabcreations** [HeidiSQL/HeidiSQL#2053]
### Description

I have a database table with both x and y overflow while viewing the contents in the Data tab.

When I click the "**_scrollbar channel_**" or "**_scrollbar trough_**" of the horizontal **_or_** vertical scrollbar (to trigger a page up/down or in this case page left/right) it inexplicably adds a column to sort.

If I click the scrollbar buttons or the **_scroll box_** or _**scroll thumb**_ the column sorted is not effected.

I've attached a couple of images to clarify the scrollbar anatomy.

![1-Figure1-1](https://github.com/user-attachments/assets/94a72f40-578d-4164-8939-22ac4130e09e)
![images](https://github.com/user-attachments/assets/1e722fdf-a6b7-4a9b-8485-dbd89e3831d1)

### HeidiSQL version

12.8.0.6908

### Database server version

MariaDB 11.4.2

### Reproduction recipe

Click the "**_scrollbar channel_**" or "**_scrollbar trough_**" of the horizontal **_or_** vertical scrollbar in a table with x and/or y overflow in the Data tab while there is no sorting. If the table now is suddenly sorting then you've reproduced the bug.

**Comment by ansgarbecker** [comment]
Your video showed you had the "Sorting" dialog open when that happens. This is probably the crux here:
![grafik](https://github.com/user-attachments/assets/912681a2-da87-40d9-92fa-6e428a651f01)

Could you please update to the latest build and retest. Build 6908 is 50 commits behind the latest build.

**Comment by jabcreations** [comment]
No, I cropped the video and showed it as being empty initially for your convenience so you could see there was no sorting set to start - or I could upload a 4GB 10 minute long video? If it's late and you're tired just come back to the issue tomorrow.

**Comment by ansgarbecker** [comment]
No it's 10 o'clock in the morning here :)
I meant the sorting dialog is open, several times, like in the above screenshot of your video. Sometimes with one sorted column, sometimes without. Nearly all of your clicks on the scrollbar are the first one _out of that dialog_. I was just thinking the opened dialog could cause some issue here. But I just see one click (at ~12s in your video) is without the dialog, so maybe there is still a different issue. Remember I am just guessing yet.
Please update to the latest build.

**Comment by jabcreations** [comment]
Again, I intentionally went out of my way to show you what was happening. Did you not reproduce the issue at your end? I downloaded and installed build 6958 and just confirmed the bug is still happening. Is it you're not able to reproduce at your end or are you just really not in the mood to deal with another bug? Remember, I'm a developer too and sometimes it gets a bit too Starship Troopers for my taste too.

**Comment by ansgarbecker** [comment]
Hey no need to be rude. I'm trying to guess what is causing the issue, nothing else. All I am asking for is whether the dialog was open while you clicked the scrollbar.

**Comment by jabcreations** [comment]
Not trying to be rude, my apologies. So what else can I do? I can ZIP the install directory and any settings (registry, appData, etc) if you want? I can also list installed dependencies if there are any with some odd version combination? Perhaps it's some oddity with the database table itself?

**Comment by ansgarbecker** [comment]
Well you have the latest build, like me. The libraries in the HeidiSQL folder are all drivers for database abstraction. So the only thing you could do is export your registry settings for me (without sensitive data, e.g. leave out the whole "Servers" tree).
Also, you could try to reproduce on a different system.

**Comment by jabcreations** [comment]
Hi Ansgar, okay no problem. I need a few days as I'm scrambling to get my Mail Module working well enough as Yahoo Mail is committing suicide. I've created a reminder in my calendar for this weekend. Thanks!

**Comment by jabcreations** [comment]
Greetings Ansgar! So I was encountering the bug again (still busy) though I had an idea: are you at 1920x1080 or any other resoluition besides 2560x1440? I did change it temporarily down to 1920x1080 and the bug still occurs.

I exported the registry and was reasonably thorough about removing what looked like private information (and the servers have been left out). I hope this helps! Thank you!

```
[HKEY_CURRENT_USER\SOFTWARE\HeidiSQL]
"ToolbarShowCaptions"=dword:00000001
"Updatecheck"=dword:00000001
"UpdatecheckLastrun"="2024-12-28 19:16:35"
"AutoReconnect"=dword:00000001
"SQL Attr Comment Foreground"=dword:007a6e54
"SQL Attr Comment Background"=dword:1fffffff
"SQL Attr Comment Style"=dword:00000002
"SQL Attr ConditionalComment Foreground"=dword:00b8c26d
"SQL Attr ConditionalComment Background"=dword:1fffffff
"SQL Attr ConditionalComment Style"=dword:00000002
"SQL Attr DataType Foreground"=dword:00ea92c7
"SQL Attr DataType Background"=dword:1fffffff
"SQL Attr DataType Style"=dword:00000001
"SQL Attr DefaultPackages Foreground"=dword:1fffffff
"SQL Attr DefaultPackages Background"=dword:1fffffff
"SQL Attr DefaultPackages Style"=dword:00000001
"SQL Attr DelimitedIdentifier Foreground"=dword:00ffb182
"SQL Attr DelimitedIdentifier Background"=dword:1fffffff
"SQL Attr DelimitedIdentifier Style"=dword:00000000
"SQL Attr Exception Foreground"=dword:1fffffff
"SQL Attr Exception Background"=dword:1fffffff
"SQL Attr Exception Style"=dword:00000002
"SQL Attr Function Foreground"=dword:00e3cec3
"SQL Attr Function Background"=dword:1fffffff
"SQL Attr Function Style"=dword:00000001
"SQL Attr Identifier Foreground"=dword:00ffb182
"SQL Attr Identifier Background"=dword:1fffffff
"SQL Attr Identifier Style"=dword:00000000
"SQL Attr Number Foreground"=dword:007053ff
"SQL Attr Number Background"=dword:1fffffff
"SQL Attr Number Style"=dword:00000000
"SQL Attr PLSQL-ReservedWord Foreground"=dword:1fffffff
"SQL Attr PLSQL-ReservedWord Background"=dword:1fffffff
"SQL Attr PLSQL-ReservedWord Style"=dword:00000001
"SQL Attr ProcName Foreground"=dword:1fffffff
"SQL Attr ProcName Background"=dword:1fffffff
"SQL Attr ProcName Style"=dword:00000000
"SQL Attr ReservedWord Foreground"=dword:00ff0000
"SQL Attr ReservedWord Background"=dword:1fffffff
"SQL Attr ReservedWord Style"=dword:00000001
"SQL Attr Space Foreground"=dword:1fffffff
"SQL Attr Space Background"=dword:1fffffff
"SQL Attr Space Style"=dword:00000000
"SQL Attr SQLPlus-Command Foreground"=dword:1fffffff
"SQL Attr SQLPlus-Command Background"=dword:1fffffff
"SQL Attr SQLPlus-Command Style"=dword:00000001
"SQL Attr String Foreground"=dword:0087e8c3
"SQL Attr String Background"=dword:1fffffff
"SQL Attr String Style"=dword:00000000
"SQL Attr Symbol Foreground"=dword:00c4cb80
"SQL Attr Symbol Background"=dword:1fffffff
"SQL Attr Symbol Style"=dword:00000000
"SQL Attr TableName Foreground"=dword:006976f7
"SQL Attr TableName Background"=dword:1fffffff
"SQL Attr TableName Style"=dword:00000000
"SQL Attr Variable Foreground"=dword:006bcbff
"SQL Attr Variable Background"=dword:1fffffff
"SQL Attr Variable Style"=dword:00000000
"FieldEditor_Set"=dword:00000001
"LineBreakStyle"=dword:00000001
"LastSessions"="Local, MariaDB; root"
"LastActiveSession"="Local, MariaDB; root"
"ColWidths_connform.ListSessions"="233,107,50,50,176,50,103"
"ColsVisible_connform.ListSessions"="0,1,4,6"
"ColPositions_connform.ListSessions"="0,1,2,3,4,5,6"
"ColSort_connform.ListSessions"="4,0"
"ColWidths_UserManagerForm.listUsers"="179,141"
"ColsVisible_UserManagerForm.listUsers"="0,1"
"ColPositions_UserManagerForm.listUsers"="0,1"
"ColSort_UserManagerForm.listUsers"="0,0"
"CoolBand0Index"=dword:00000000
"CoolBand0Break"=dword:00000001
"CoolBand0Width"=dword:0000077a
"CoolBand1Index"=dword:00000001
"CoolBand1Break"=dword:00000001
"CoolBand1Width"=dword:0000077a
"MainWinMaximized"=dword:00000001
"MainWinOnMonitor"=dword:00000000
"ColWidths_MainForm.ListDatabases"="313,80,50,50,50,50,50,50,50,50,237"
"ColsVisible_MainForm.ListDatabases"="0,1,2,3,4,5,6,7,8,9,10"
"ColPositions_MainForm.ListDatabases"="0,1,2,3,4,5,6,7,8,9,10"
"ColSort_MainForm.ListDatabases"="0,0"
"ColWidths_MainForm.ListVariables"="268,200,208"
"ColsVisible_MainForm.ListVariables"="0,1,2"
"ColPositions_MainForm.ListVariables"="0,1,2"
"ColSort_MainForm.ListVariables"="0,0"
"ColWidths_MainForm.ListStatus"="160,316,100,100"
"ColsVisible_MainForm.ListStatus"="0,1,2,3"
"ColPositions_MainForm.ListStatus"="0,1,2,3"
"ColSort_MainForm.ListStatus"="0,0"
"ColWidths_MainForm.ListProcesses"="70,80,80,80,80,50,50,10"
"ColsVisible_MainForm.ListProcesses"="0,1,2,3,4,5,6,7"
"ColPositions_MainForm.ListProcesses"="0,1,2,3,4,5,6,7"
"ColSort_MainForm.ListProcesses"="0,1"
"ColWidths_MainForm.ListCommandStats"="120,100,100,100,256"
"ColsVisible_MainForm.ListCommandStats"="0,1,2,3,4"
"ColPositions_MainForm.ListCommandStats"="0,1,2,3,4"
"ColSort_MainForm.ListCommandStats"="1,1"
"ColWidths_MainForm.ListTables"="308,70,121,120,301,131,100,50,70,70,70,70,70,90,120,259,70,70,83"
"ColsVisible_MainForm.ListTables"="0,1,2,3,4,5,6,15,18"
"ColPositions_MainForm.ListTables"="0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18"
"ColSort_MainForm.ListTables"="0,0"
"dbtreewidth"=dword:000001e4
"PlinkExecutable"="C:\\MEDIA\\INTERNET\\putty.exe"
"querymemoheight"=dword:00000127
"sqloutheight"=dword:00000131
"Shortcut1_actDropObjects"=dword:0000602e
"ColWidths_frmTableEditor.listColumns"="43,323,156,90,73,95,50,175,130,230,100,100,50"
"ColsVisible_frmTableEditor.listColumns"="0,1,2,3,4,5,6,7,8,9,10,11"
"ColPositions_frmTableEditor.listColumns"="0,1,2,3,4,5,6,7,8,9,10,11,12"
"ColSort_frmTableEditor.listColumns"="-1,0"
"ColWidths_frmTableEditor.treeIndexes"="434,100,80,120,80"
"ColsVisible_frmTableEditor.treeIndexes"="0,1,2"
"ColPositions_frmTableEditor.treeIndexes"="0,1,2,3,4"
"ColSort_frmTableEditor.treeIndexes"="-1,0"
"ColWidths_frmTableEditor.listForeignKeys"="200,80,100,80,80,80"
"ColsVisible_frmTableEditor.listForeignKeys"="0,1,2,3,4,5"
"ColPositions_frmTableEditor.listForeignKeys"="0,1,2,3,4,5"
"ColSort_frmTableEditor.listForeignKeys"="-1,0"
"Shortcut1_actExecuteQuery"=dword:0000400d
"MemoEditorWidth"=dword:0000090f
"MemoEditorHeight"=dword:000004b3
"GridExportEncoding"=dword:00000004
"GridExportSelection"=dword:00000000
"GridExportOutputCopy"=dword:00000000
"GridExportOutputFile"=dword:00000001
"GridExportFilename"="[Nope]"
"GridExportRecentFiles"="[Nope]"
"GridExportFormat"=dword:00000004
"FindDialogSearchHistory"="[Nope]"
"DateTimeEditor_CursorPos_Type4"=dword:0000000b
"TableEditorTabsHeight"=dword:000000e4
"ExportSQL_CreateTables"=dword:00000001
"ExportSQL_DataHow"=dword:00000002
"ExportSQL_Filenames"="[Nope.]"
"Usermanager_WindowWidth"=dword:0000041b
"Usermanager_WindowHeight"=dword:00000574
"Usermanager_ListWidth"=dword:00000144
"TabWidth"=dword:00000001
"DataLocalNumberFormat"=dword:00000000
"TabsToSpaces"=dword:00000001
"FileImport_WindowWidth"=dword:00000328
"FileImport_WindowHeight"=dword:00000244
"ColWidths_frmInsertFiles.ListColumns"="100,80,556"
"ColsVisible_frmInsertFiles.ListColumns"="0,1,2"
"ColPositions_frmInsertFiles.ListColumns"="0,1,2"
"ColSort_frmInsertFiles.ListColumns"="-1,0"
"ColWidths_frmInsertFiles.ListFiles"="603,70,80"
"ColsVisible_frmInsertFiles.ListFiles"="0,1,2"
"ColPositions_frmInsertFiles.ListFiles"="0,1,2"
"ColSort_frmInsertFiles.ListFiles"="0,0"
"FileDialogEncoding_MainForm"=dword:00000000
"ExportSQL_CreateDatabases"=dword:00000000
"ExportSQL_Output"=dword:00000003
"TableTools_TreeWidth"=dword:000001b2
"DoUsageStatistics"=dword:00000001
"Theme"="Windows10 Dark"
"SQL Attr Key Foreground"=dword:00e3cec3
"FieldColor_Binary"=dword:00c9767f
"FieldColor_Datetime"=dword:007373c9
"FieldColor_Numeric"=dword:00c0c0c0
"FieldColor_Other"=dword:0073c1c1
"FieldColor_Real"=dword:00d07d7d
"FieldColor_Spatial"=dword:00cece73
"FieldColor_Text"=dword:0073d573
"LastUsageStatisticCall"="2024-12-24 20:09:02"
"SessionManager_ListWidth"=dword:0000026f
"SessionManager_WindowWidth"=dword:000005ad
"SessionManager_WindowHeight"=dword:00000286
"SessionManager_WindowLeft"=dword:000000c9
"SessionManager_WindowTop"=dword:000000ed
"SQL Attr ConsoleOutput Foreground"=dword:1fffffff
"SQL Attr ConsoleOutput Background"=dword:1fffffff
"SQL Attr ConsoleOutput Style"=dword:00000005
"SQL Attr Key Background"=dword:1fffffff
"SQL Attr Key Style"=dword:00000001
"SQLColMatchingBraceForeground"="$0028EFFF"
"SQLColMatchingBraceBackground"="$004D513B"
"IconPack"="Icons8"
"GUIFontName"="@Malgun Gothic"
"GUIFontSize"=dword:0000000c
"MainWinLeft"=dword:fffffffc
"MainWinTop"=dword:fffffffc
"MainWinWidth"=dword:000003d9
"MainWinHeight"=dword:00000204
"TableTools_WindowWidth"=dword:00000466
"TableTools_WindowHeight"=dword:00000225
"FontSize"=dword:00000010
"FavoriteObjectsOnly"=dword:00000001
"ColWidths_frmTableEditor.listCheckConstraints"="200,420"
"ColsVisible_frmTableEditor.listCheckConstraints"="0,1"
"ColPositions_frmTableEditor.listCheckConstraints"="0,1"
"ColSort_frmTableEditor.listCheckConstraints"="-1,0"
"CreateDbCollation"="utf8mb4_unicode_520_ci"
"Shortcut1_ecDeleteLine"=dword:00000000
"Shortcut1_ecRedo"=dword:0000c05a
"ColumnSelectorWidth"=dword:000000c9
"GridExportWindowWidth"=dword:00000213
"LogDebug"=dword:00000001
"PreferencesWindowWidth"=dword:00000316
"PreferencesWindowHeight"=dword:0000027d
"ThemePreviewTop"=dword:00000191
"ThemePreviewLeft"=dword:0000021b
"SelectDBO_WindowWidth"=dword:000001d7
"SelectDBO_WindowHeight"=dword:00000153
"TableTools_FindText"="jabcreat_v32setup"
"DoubleClickInsertsNodeText"=dword:00000000
"SQLFile0"="[Nope]"
"SQLFile1"="[Nope]"
"SQLFile2"="[Nope]"
"SQLFile3"="[Nope]"
"SQLFile4"="[Nope]"
"SQLFile5"="[Nope]"
"SQLFile6"="[Nope]"
"SQLFile7"="[Nope]"
"SQLFile8"="[Nope]"
"SQLFile9"="[Nope]"
"SQLFile10"="[Nope]"
"SQLFile11"="[Nope]"
"SQLFile12"="[Nope]"
"SQLFile13"="[Nope]"
"SQLFile14"="[Nope]"
"SQLFile15"="[Nope]"
"SQLFile16"="[Nope]"
```

**Comment by ansgarbecker** [comment]
Only minor differences in your settings and mine if I compare them. Only things I stumbled upon:
* GUIFontName is "@Malgun Gothic" - a Korean font series (unlikely to make problems here but who knows)
* ToolbarShowCaptions is activated
* Theme is "Windows10 Dark"

I tried again to reproduce the issue by clicking the scrollbar "trough" on all Windows systems I have by my hands here, using HeidiSQL build 6978 and Theme set to "Windows 10 Dark":
* Windows 11 Pro at 100% DPI and 1920x1080 px
* Windows 11 Pro at 125% DPI and 2560x1440 px
* Windows Server 2016 at 100% DPI and 1280x1024 px
* Windows 10 Enterprise in HyperV at 125% DPI and 1280x1024 px

None of these tests applied a sorting after click on the scrollbar trough. So I'm still unable to fix something here.

I also inspected events in the code where a sorting gets applied. There are three of them:
* click on the sort dialog + click on "add new" button
* click on grid column header
* refreshing grid and applying the previously used sortings

None of these is related to the scrollbar.

**Comment by jabcreations** [comment]
I'm about to lay down though I had a thought. Earlier today I noticed that when I went to click the "Delete Selected Rows" button it was attempting to insert a row. Well, the button to the right is the "Insert Row" button. I prefer larger text. So the "Insert Row" button was overflowing over the first one.

>>
    GUIFontName is "@malgun Gothic" - a Korean font series (unlikely to make problems here but who knows)
    ToolbarShowCaptions is activated
    Theme is "Windows10 Dark"

I set these to what I guess are the defaults and that had no effect.

Try jacking the font size up to like 20+. I just set it down to 7 though that also had no effect (after restarting the program). My DPI is 100%.

So when I'm having GUI problems I basically use CSS outline: #f0f solid 2px; on everything to visualize things that might not appear. If you've got some way for me to apply such an effect I'd be happy to take some screenshots.

**Comment by ansgarbecker** [comment]
Good idea. I just pushed a change so the next build shows debug lines when any of the events fire, including the name of the event so we can trace it back then.
So, once you updated to the next build, please
* activate debug logging, in Tools > Preferences > Logging
* on the same page, make sure "Log last lines" is set to at least 5000
* apply settings
* try to reproduce the scrollbar issue, without too many clicks, so the log does blow up with useless details
* once you saw the sorting issue, copy the whole contents of the log panel, paste it into  Notepad or whatever and search for lines containing the text "_sorting for column_":
![grafik](https://github.com/user-attachments/assets/cc395aaf-5595-4ec5-acb3-dd72e8476a53)

**Comment by jabcreations** [comment]
Thank you Ansgar! So I sent you an email with the output (since it involves my internal/private database structure) to your anse email address. Hopefully I followed your directions correctly. Please let me know if you need anything else from me to help you conquer this bug.

Bugs!!!!
![bugs!!!!](https://github.com/user-attachments/assets/a210526a-ed87-4cf1-b841-3c7d5cb8a665)

**Comment by ansgarbecker** [comment]
The first part of your debug lines tell me you clicked the header of column "headers_from", and Heidi added that on top of 3 previously sortings. So did you indeed click the column header or is that from the scrollbar-click?
```
/* Created sorting for column headers_from/0 in TMainForm.DataGridHeaderClick */
/* Ping server ... */
/* Restored sorting for column error_server_log/1 in TMainForm.HandleDataGridAttributes */
/* Restored sorting for column headers/0 in TMainForm.HandleDataGridAttributes */
/* Restored sorting for column id_mail_messages_replied/0 in TMainForm.HandleDataGridAttributes */
/* Restored sorting for column headers_from/0 in TMainForm.HandleDataGridAttributes */
```

**Comment by jabcreations** [comment]
No, as previously stated, I explicitly clicked on the "scrollbar trough" or "scrollbar channel" as illustrated in an earlier post.

Could this possibly be linked to the GUI invisibly resizing due to changes such as the font size?

In example with the text labels displayed for the toolbar I found myself clicking on the "Insert row" button even though I was explicitly clicking on the "Delete select rows" button. I imagine that the button text overflows the button's borders. However in this bug I'm clicking on the trough or channel at the bottom scrollbar to quickly horizontally scroll.

Is there a way I can instantly add an outline to all of the GUI elements that are on screen and then send you a screenshot?

**Comment by ansgarbecker** [comment]
> Could this possibly be linked to the GUI invisibly resizing due to changes such as the font size?

Well you click the scrollbar but the GUI gets a click event on the column header. I never saw that happening, neither due to a font size or with other causes. But I recall there were a few related issues when external mouse or keyboard drivers/tools were being actively running. Can't recall them right now but I believe removing that external tool solved the issue then.

> Is there a way I can instantly add an outline to all of the GUI elements that are on screen and then send you a screenshot?

No, there is no way to do that. Note this is not CSS/HTML.

Did you ever see that issue on a different system, or is it only happening on one? Would definitely be good you try that on a second (or third) system.

**Comment by jabcreations** [comment]
The very first thing I did fixed the issue. My apologies for the wild goose chase.

I am currently using WindowBlinds which I use to impose full dark mode on Windows 10:

https://www.stardock.com/products/windowblinds/

I am using a version of the Fl8t theme that I modified and made available here:

https://www.wincustomize.com/explore/windowblinds/9662

It does override the default system colors very effectively...so effectively that lazy web designers who don't set default background-color or color properties force me to have to "fix" their website one way or another (usually through the Stylus extension).

But that is for colors. I've never see anything in any other program suggest that it moves GUI components around short of the window control buttons. In example for the Fl8t theme I had to fix the buttons so the close button was at the absolute top-right, otherwise I have to stop and think about moving the mouse away from the corner which is retarded but people who are blindly in to aesthetics often have poor  (if any) comprehension about functionality.

I tested about five other themes and they all introduced the same bug.  I'm running version 10.89 which was released several years ago. I'll reach out to their people and link them to this thread. I don't really feel like dropping $30 for an upgrade to maybe fix one obnoxious bug. I'm also in no big rush (or, any) to "upgrade" to Windows 11 on my desktop with all the stupid things they keep doing to the OS. Maybe with a, hopefully competent/helpful, support staff member I can get some answers.

**Comment by jabcreations** [comment]
SOLVED!

So I got curious and typed "windowBlinds" "HeidiSQL" in to Google for a quick check and came across this thread:

https://www.heidisql.com/forum.php?t=607

Well, I added HeidiSQL as an exclusion in WindowBlinds, exited HeidiSQL, reapplied the theme I use and started HeidiSQL back up. I clicked on the scrollbar to trigger the bug and the bug wasn't triggered! Wow, apparently we can have happy endings in the real world! 🙂︀

I should apply exceptions to my regular browsers so I don't have to fix websites for lazy web designers. Again, my sincere apologies for the wild goose chase and my gratitude for your efforts to help me figure out what the problem was.

**Comment by ansgarbecker** [comment]
I should probably add a note to the compatibility section on the download page...

**Comment by jabcreations** [comment]
I checked the downloads page and you already added it, nice.

Mind if I inquire what compatibility issues there are with Windows 10? Perhaps you could link the compatibility notes to various pages with further notes? If you throw something together here is a screenshot so people who come across it can figure things out quickly enough.

![windowsblinds-exceptions](https://github.com/user-attachments/assets/134355b8-7140-4acb-9320-2f1a5ba280f2)

**Comment by ansgarbecker** [comment]
No specific issues with Windowas 10, but the impression from the myriad of issues I read about in the last year. And probably due to the fact Windows 10 runs out this year.

I don't want to provide more details about third party software. The short note should be enough, I'm not responsible for their issues.

**Comment by jabcreations** [comment]
Fair enough. I'm on 22H2. From what I've gathered Microsoft is intentionally sabotaging Windows 10 builds to "encourage" people to install Windows 11. We're done, we're good and Happy New Year!

**Comment by babanga** [comment]
> Well you click the scrollbar but the GUI gets a click event on the column header. I never saw that happening

Always had it. And it bothered me for years slowing down the query with extra sortings, especially when you didn't notice it.

![Image](https://github.com/user-attachments/assets/2200e59b-88de-4d3b-91cd-f0349a292c33)
